index.html

```diff
@@ -11,5 +11,87 @@
     <header class="app-title">
       <h1>Quiz App</h1>
     </header>
+    <main>
+      <section class="quiz-card">
+        <img
+          src="img/bookmark-regular.svg"
+          alt="Bookmark Icon"
+          class="quiz-bookmark"
+        />
+        <article class="quiz-question">What is Lorem?</article>
```

Comment:
You could change this to a heading. Which is semantically better, since a section always contains a heading

```suggestion
        <h2 class="quiz-question">What is Lorem?</h2>
```